docs/prow/workload-clusters.md

```diff
@@ -0,0 +1,60 @@
+# Prow workload clusters
+
+It's a description of prow workload clusters. These are clusters where prow is scheduling pods which execute prowjob logic. All workload clusters are aggregated under kyma-prow GCP project. We use two workload clusters for trusted and untrusted prowjobs.
+
+## Clusters design
+
+Clusters have enabled autoupgrade and follow stable channel.
+Clusters have enabled autoscaling for all node pools.
+Clusters are private with unrestricted access to k8s API from Internet domain.
+Clusters use CloudNAT and Private Google Access.
+Clusters are regional.
+Clusters use separate subnets for nodes, pods and services.
+
+```gcloud container clusters list
+   NAME                          LOCATION        MASTER_VERSION  MASTER_IP       MACHINE_TYPE   NODE_VERSION    NUM_NODES  STATUS
+   trusted-workload-kyma-prow    europe-west3    1.14.10-gke.36  _____________   n1-standard-4  1.14.10-gke.36  3          RUNNING
+   untrusted-workload-kyma-prow  europe-west3    1.14.10-gke.36  _____________   n1-standard-4  1.14.10-gke.36  2          RUNNING
+```
+
+## Infrastructure design
+
+Cluster are located in separate networks for trusted and untrusted components. Each networks provide three subnets for cluster nodes, pods and services.
+There is no peering between networks, thus clusters are isolated on network level.
+Each cluster has dedicated Cloud Router with CloudNAT and external IP. This provide outgoing connectivity for clusters and fixed external IP from which all traffic is seen.
```

Comment:
```suggestion
Each cluster has a dedicated Cloud Router with Cloud NAT and an external IP address. This provides outgoing connectivity for clusters and a fixed external IP address that receives all traffic.
```

Reply:
This IP is not receiving all traffic. It's used as a source IP for all traffic initiated from pods or worker nodes. What about, ".. IP address from which, all traffic originated from pods and worker nodes is seen."?

Reply:
Ok, I just changed passive voice into active voice - see my updated suggestion above :)